pkg/minikube/kubeconfig/config_test.go

```diff
@@ -33,7 +33,7 @@ clusters:
 - cluster:
     certificate-authority: /home/la-croix/apiserver.crt
     server: 192.168.1.1:8080
-  name: la-croix
+  name: minikube
```

Comment:
We should probably handle the case where there is no minikube entry in .kube/config as it possible a user removed this or no entry was added even though minikube is running